src/views/HeaderBackButton.js

```diff
@@ -39,14 +53,27 @@ const styles = StyleSheet.create({
   container: {
     alignItems: 'center',
     justifyContent: 'center',
+    flexDirection: 'row',
   },
-  button: {
-    height: 24,
-    width: 24,
-    margin: Platform.OS === 'ios' ? 10 : 16,
-    resizeMode: 'contain',
-    transform: [{ scaleX: I18nManager.isRTL ? -1 : 1 }],
+  backButton: {
+    fontSize: 17,
   },
+  button: Platform.select({
```

Comment:
This will break on windows, not sure if we support it yet, but better to keep in mind.